audb/core/api.py

```diff
@@ -261,6 +261,8 @@ def dependencies(
     deps_path = os.path.join(db_root, define.CACHED_DEPENDENCIES_FILE)
 
     deps = Dependencies()
+    deps._name = name
```

Comment:
This is not really nice. Maybe we should make `_filter_media()`, `_filter_tables()` and `_error_message_missing_object`  helper functions in `dependencies.py` but not class methods of `Dependencies`? As far as I see they are not exposed to the user and never called within `Dependencies`.

Reply:
I switched to implement them as helper functions now.